presto-main/src/main/java/io/prestosql/Session.java

```diff
@@ -767,7 +767,10 @@ public ResourceEstimateBuilder setPeakMemory(DataSize peakMemory)
 
         public ResourceEstimates build()
         {
-            return new ResourceEstimates(executionTime, cpuTime, peakMemory);
+            return new ResourceEstimates(
+                    executionTime.map(value -> java.time.Duration.ofMillis(value.toMillis())),
+                    cpuTime.map(value -> java.time.Duration.ofMillis(value.toMillis())),
```

Comment:
We briefly discussed adding converters to units `Duration`, as that seems obvious at first, but there are some design questions:

* When going to JDK Duration, how much precision do we use? Should we always do full nanosecond precision, even though this might be "inventing" values due to floating point conversion errors?
* When going from JDK duration, what unit and precision do we use?